src/data/staking-products.json

```
@@ -20,7 +20,7 @@
           "url": "https://github.com/trailofbits/publications/blob/master/reviews/RocketPool.pdf"
         }
       ],
-      "minEth": 16,
+      "minEth": 8,
```

```suggestion
      "minEth": 10.4,
```

Going to change this to 10.4 as you need to have RPL as well will RocketPool. 

On their node operators page they also highlight 10.4 due to this requirement: https://rocketpool.net/node-operators